attendance/templates/attendance_base.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+{% load static %}
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="//cdn.muicss.com/mui-0.9.43/css/mui.min.css" rel="stylesheet" type="text/css" />
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link rel="stylesheet" href="{% static  'css/mui_assets.css' %}"/>
+    <link rel="icon" href="https://github.com/Ernesto2004/MB_Portal_Design/blob/master/mblogo.png?raw=true"> <!-- LOGO ON THE TAB BAR -->
+    <!-- MDL STUFF -->
+    <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
+    <!-- -->
+    <script src="//cdn.muicss.com/mui-0.9.43/js/mui.min.js"></script>
+    <script src="//code.jquery.com/jquery-2.1.4.min.js"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <title>Mission Bit</title>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/instafeed.js/1.4.1/instafeed.min.js"></script>
+     <script type="text/javascript">
+        var galleryFeed = new Instafeed({
+        get: "user",
+        userId: "2052224602",
+        accessToken: '2052224602.1677ed0.1292ba7a40024b4091eb308010d4ac7e',
+        resolution: "standard_resolution",
+        useHttp: "true",
+        limit: 5,
+        target: "instafeed",
+        template: '<h4 class="slot-description">\{\{location\}\}</h4>' +
+            '<a class="instafeedimg" href="\{\{image\}\}" target="_blank"><img src="\{\{image\}\}"></a>' +
+            '<h4 class="slot-description"></h4><h4 class="slot-description"><img src="https://github.com/Ernesto2004/MB_Portal_Design/blob/master/heart-icon.png?raw=true" width="30" height="30" style="opacity: 0.75">\{\{likes\}\} ' +
```

Comment:
These image assets should be local URLs, you can use `{% static … %}` to get the appropriate URLs rather than hard-coding them.

Reply:
Agreed, will address

Reply:
Taken care of by Ernesto